dlgr/griduniverse/experiment.py

```diff
@@ -1518,7 +1529,7 @@ def average_payoff(self, data):
         dataState = df.loc[df['type'] == 'state']
         if dataState.empty:
             return 0.0
-        final_state = json.loads(dataState.iloc[-1][-2])
+        final_state = json.loads(dataState.iloc[-1][-1])
```

Comment:
This line along with 1542 had "buffer errors" for incorrect indexing it seemed. I recall this used to be [-1]. I tested it and it worked, so I reverted back to the earlier [-1]. However, I am a little confused what changed. I'll take a closer look at this data frame.